identity_credential/src/credential/revocation_bitmap_status.rs

```diff
@@ -19,14 +19,37 @@ use crate::error::Result;
 pub struct RevocationBitmapStatus(Status);
 
 impl RevocationBitmapStatus {
-  const INDEX_PROPERTY_NAME: &'static str = "revocationBitmapIndex";
+  const REVOCATION_BITMAP_INDEX_PROPERTY: &'static str = "revocationBitmapIndex";
```

Comment:
Nit: bit redundant since it's accessed as `RevocationBitmapStatus::REVOCATION_BITMAP_INDEX_PROPERTY`, whereas before it was `RevocationBitmapStatus::INDEX_PROPERTY_NAME`, but explicitness is probably preferred.